specification/attributes/amortization_handling.md

```diff
@@ -0,0 +1,34 @@
+# Amortization Handling
+
+Amortization is a process used to break down and spread purchase costs over the period of time or term of use. 
+
+Amortization is primarily used in purchasable commitment-based discounts. When a purchase is applicable to resources, like commitment-based discounts, the amortized cost of a resource takes the initial payment and term into account and distributes it out based on resource usage, attributing the prorated cost for each hour of billing. Amortization provides a method for FinOps practitioners to reallocate significant purchase charges to the appropriate audience in support of their cost allocation efforts. 
+
+## Attribute ID
+
+AmortizationHandling
+
+## Attribute name
+
+Amortization Handling
+
+## Description
+
+Rules and requirements defining how amortization is shown in the billing data.
+
+## Requirements
+
+* Every charge MUST include all applicable discounts. (e.g., negotiated and commitment-based discounts)
+* Discounts SHOULD be applied directly to the charges they are discounting.
+* All purchases SHOULD be amortized.
```

Comment:
When an unused value is rolled over from one contract to a subsequent contract - is that an exception to the "MUST" criteria? I'm familiar with a pre-paid commitment with a 10% rollover clause; in practice - if the customer uses 92% of what they prepaid - the remaining 8% is added to the subsequent contract - but does NOT have to be purchased again.
This would get complicated in the scenario where a customer consumes 85% of what they pre-paid, rolls over 10% to a subsequent contract and has 5% that they have to add into their cost data.
Happy to work through scenarios if you believe it needs to be part of V1.0

Reply:
This has been refactored in the latest version. Can you take a look? Resolving this for now. Please leave any comments if you notice anything.